src/main/java/org/wso2/lsp4intellij/contributors/annotator/LSPAnnotator.java

```diff
@@ -46,20 +46,24 @@
 import java.util.ConcurrentModificationException;
 import java.util.HashMap;
 import java.util.List;
+import java.util.Objects;
 
 public class LSPAnnotator extends ExternalAnnotator<Object, Object> {
 
     private static final Logger LOG = Logger.getInstance(LSPAnnotator.class);
     private static final Object RESULT = new Object();
-    private static final HashMap<DiagnosticSeverity, HighlightSeverity> lspToIntellijAnnotationsMap = new HashMap<>();
+    private static final HashMap<DiagnosticSeverity, HighlightSeverity> annotationsMap = new HashMap<>();
```

Comment:
```suggestion
    private static final Map<DiagnosticSeverity, HighlightSeverity> annotationsMap = new HashMap<>();
```
Do we need add it like this?

Reply:
+1. Updated with https://github.com/ballerina-platform/lsp4intellij/pull/384/commits/5c82a152524dbbf63510b11ca351b9da880353aa